query-engine/connectors/mongodb-query-connector/src/root_queries/mod.rs

```diff
@@ -69,7 +71,9 @@ where
     let span = info_span!(
         "prisma:engine:db_query",
         user_facing = true,
-        "db.statement" = %Arc::clone(&query_string)
+        "db.system" = SYSTEM_NAME,
```

Comment:
By the way, I only just realized that Mongo has separate conventions from SQL databases: https://opentelemetry.io/docs/specs/semconv/database/mongodb/

Aside from `db.system`, another required one is `db.collection.name` (and `db.operation.name` is also conditionally required if readily available).

You can get  both of them here by making the corresponding method in the `QueryString` trait `pub(crate)` and calling them here as `builder.collection()` and `builder.query_type()`.